.make/test.mk

```diff
@@ -332,7 +332,7 @@ $(eval COMBINED_OUT_FILE := $(3))
 $(eval ERRORS_FILE := $(4))
 $(eval ENV_VAR := $(5))
 $(eval ALL_PKGS_COMMA_SEPARATED := $(6))
-
+@printf "\033[36mTesting $(2)\033[0m\r\n"
```

Comment:
Please wrap coloring in a separate function so it can be disabled in cases where no coloring is enabled. Maybe check for some environment variable.